### ISSUE TITLE
DSG handling of JSON content

### ISSUE THREAD

**Issue by JohnMoehrke** [IHE/IT-Infrastructure#195]
https://github.com/IHE/IT-Infrastructure/blob/master/Proposals/IHE_Profile_Proposal_Template-jDSG.docx

**Comment by JohnMoehrke** [comment]
Given that DSG covers use-cases but uses XML-Signature for the signature document
When a product or organization wants to use JSON signature documents

Update DSG profile with an option to use JSON signature as the signature document. Otherwise everything else stays the same. That is to use external signature document with an XDS relationship to the signed document, where the signature is over the signed document as a binary object.

**Comment by JohnMoehrke** [comment]
I could help with the changes, but don't have the json signature expertise. We need an expert on json signature.

**Comment by JohnMoehrke** [comment]
yes, I spoke with @litlfred , and that discussion was indicating the gap was the signature format, not the object being signed... I do agree that the proposal does look more like wanting to sign natively json objects.

**Comment by JohnMoehrke** [comment]
current DSG is content agnostic... meaning it signs the document that it is signing as a binary blob. So it can already sign a JSON document, such as a FHIR-Document encoded in JSON.

**Comment by litlfred** [comment]
Confirming that it is intended to be the signature format itself.   The
reference to FHIR json documents was more of thinking about keeping things
in the same tool chain (there may be a preference to have a JSON signature
for a JSON document rather than all the sudden having to load up an XML
stack)

Cheers,
-carl

**Comment by litlfred** [comment]
[IHE_Profile_Proposal_Template-jDSG.docx](https://github.com/IHE/IT-Infrastructure/files/10754288/IHE_Profile_Proposal_Template-jDSG.docx)
updated proposal is here.....

**Comment by JohnMoehrke** [comment]
WHO is looking for support in augmenting the IHE DSG profile to support JSON signatures. Expertise with JAdES is needed. Please see the attached Request for Quotation and refer to https://www.who.int/about/accountability/procurement/principles-and-processes for additional guidance. Responses should be sent to [leitnerc@who.int](mailto:leitnerc@who.int) by Monday February 27. rfp-fhir-doucments-in-trust-architecture.docx

p.s. I asked, and there's no geographical or citizenship restrictions?

**Comment by JohnMoehrke** [comment]
[rfp fhir doucments in trust architecture.docx](https://github.com/IHE/IT-Infrastructure/files/10795921/rfp.fhir.doucments.in.trust.architecture.docx)

**Comment by JohnMoehrke** [comment]
> [IHE_Profile_Proposal_Template-jDSG.docx](https://github.com/IHE/IT-Infrastructure/files/10754288/IHE_Profile_Proposal_Template-jDSG.docx) clarifications in an updated proposal is here.....

The proposal in the IT-Infrastructure repo has been updated.

**Comment by lukeaduncan** [comment]
https://docs.google.com/document/u/0/d/1qgkbFfBUTKSBlY2vCq0I13j8JhTVADNL/edit?pli=1

**Comment by JohnMoehrke** [comment]
https://github.com/IHE/ITI.DSGj
